templates/upload.html

```diff
@@ -30,7 +30,7 @@ <h1>Customer Lifetime Value Predictor.</h1>
 
```

Comment:
Perhaps can change the `span` tag to `p` tag in line 27 to make use of the line height?

```
<p><b>Drag & drop your CSV file here</b><br/>or click to select a file</p>
```